compiler/src/java_plugin/cpp/java_generator.cpp

```diff
@@ -541,7 +541,7 @@ static void PrintStub(
     } else {
       p->Print(
           *vars,
-          "public static class $stub_name$ extends $AbstractStub$<$stub_name$> {\n");
+          "public static final class $stub_name$ extends $AbstractStub$<$stub_name$> {\n");
```

Comment:
deprecated stub apis should still be without `final` keyword as some users are using mockito.